src/modules/token/fungible/utils/lsk.js

```diff
@@ -10,9 +10,22 @@ BigNumber.config({ ERRORS: false });
  * @param {Strong|Number} value - Value in Beddow
  * @returns {BigNumber} Value converted to LSK
  */
-export const fromRawLsk = value => (
-  new BigNumber(value || 0).dividedBy(new BigNumber(10).pow(8)).toFixed()
-);
+export const fromRawLsk = (value) =>
+  new BigNumber(value || 0).dividedBy(new BigNumber(10).pow(8)).toFixed();
+
+/**
+ * Converts a given token amount to its token symbol denom
+ *
+ * @param {BigNumber|number} amount - Amount value to be converted
+ * @param {BigNumber|number} token - Token value merged with its equivalent metadata
+ * @returns {BigNumber} Amount value converted to the token symbol's denom
+ */
+export const changeDemnom = (amount, token = {}) => {
+  const { decimals = 8 } =
+    token.denomUnits?.find?.(({ denom }) => denom === token.symbol.toLowerCase()) || {};
+
+  return new BigNumber(amount || 0).dividedBy(new BigNumber(10).pow(decimals)).toFixed();
```

Comment:
define the constant BASE = 10;

```suggestion
  return new BigNumber(amount || 0).dividedBy(new BigNumber(BASE).pow(decimals)).toFixed();
```